server/server.py

```diff
@@ -1,4 +1,3 @@
-import os
```

Comment:
I should have mentioned on my last comment that the `os` package is needed (see line 27) 😐